packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -554,10 +562,11 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
       }
       let result = '';
       if (tagsValue.length === 0) {
-        // no tags in the string, htmlTextValue is a string
+        // no tags in the string, newValue is a result string
```

Comment:
Maybe change into `If no tags in the string, use newValue as a result string`?

Reply:
Updated as discussed